proposed-charter.txt

```diff
@@ -92,6 +92,8 @@ Other Deliverables
 
 A comprehensive test suite for all features of a specification is necessary to ensure the specification's robustness, consistency, and implementability, and to promote interoperability between User Agents. Therefore, each specification must have a companion test suite, which should be completed by the end of the Last Call phase, and must be completed, with an implementation report, before transition from Candidate Recommendation to Proposed Recommendation. Additional tests may be added to the test suite at any stage of the Recommendation track, and the maintenance of a implementation report is encouraged.
 
+In particular, since WebRTC deals with communications, specific attention will be brought to the interoperability testing of the WebRTC API implementations (both browser-to-browser and browser-to-native).
+
```

Comment:
My understanding from the discussion was that we would make sure that WebRTC browsers were able to successfully interoperate with WebRTC protocols implemented by non-browser implementations; I don't think there was specific expectations that the test harness would be built to support native gizmos, but rather that some of our test cases that would be built to test P2P interoperation would be built in a way that would allow for one of the peer to not be a browser.

With all that said, "specific attention" is pretty far from "requiring".